src/_gettsim_tests/test_piecewise_polynomial.py

```diff
@@ -0,0 +1,49 @@
+"""
+Tests for `piecewise_polynomial`
+"""
+import numpy as np
+from _gettsim.piecewise_functions import (
+    get_piecewise_parameters,
+)
+
+
+def test_get_intercepts_for_piecewise_polynomial():
+    params = {
+        "type": "piecewise_quadratic",
```

Comment:
```suggestion
        "type": "piecewise_linear",
```

Or did I miss something?